proto/rill/admin/v1/api.proto

```diff
@@ -1669,6 +1682,23 @@ message SetOrganizationMemberUserRoleRequest {
 
 message SetOrganizationMemberUserRoleResponse {}
 
+message GetOrganizationMemberUserAttributesRequest {
+  string org = 1 [(validate.rules).string.min_len = 1];
+  string email = 2 [(validate.rules).string.email = true];
+}
+
+message GetOrganizationMemberUserAttributesResponse {
+  google.protobuf.Struct attributes = 1;
+}
```

Comment:
Make this `GetOrganizationMemberUser` instead and return an `OrganizationMemberUser` value (which now has the attributes).